packages/cli/src/commands/prisma.ts

```diff
@@ -23,18 +24,29 @@ export const runPrisma = async (args: string[], silent = false) => {
   }
 
   const cp = require("cross-spawn").spawn(prismaBin, args, {
-    stdio: silent ? "ignore" : "inherit",
+    stdio: silent ? "pipe" : "inherit",
```

Comment:
This function takes one parameter named `silent`.

Before the changes, this parameter suppresses the stderr and stdout at all. Also, the function returns a boolean that represents the status of the execution.

Now, when `silent` is `true`, the stderr is captured and saved, and the function returns an object with two keys:
\- success: This represents the execution status (the original return of the function).
\- stderr: the captured stderr, that only applies when `silent` is `true`.